bundles/net.tourbook/src/net/tourbook/conconi/ConconiView.java

```diff
@@ -678,25 +678,25 @@ private void createUI_30_Test(final Composite parent) {
             //		container.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_CYAN));
             {
                // label: heartbeat value
-               _lblDeflactionPulse = new Label(containerValues, SWT.TRAIL);
+               _lblDeflectionPulse = new Label(containerValues, SWT.TRAIL);
```

Comment:
@wolfgang-ch 
I didn't rename the ` label.setText(Messages.Conconi_Chart_DeflactionPoint);` yet because translators are working on it. I will rename it after 23.8